File: VB/E4434/CustomGrid/UnboundGridViewInfoRegistrator.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraGrid.Views.Base
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Registrator

Namespace CustomGrid.CustomGrid
	Public Class UnboundGridViewInfoRegistrator
		Inherits GridInfoRegistrator
		Public Overrides ReadOnly Property ViewName() As String
			Get
				Return "UnboundGridView"
			End Get
		End Property

		Public Overrides Function CreateView(ByVal grid As GridControl) As BaseView
			Return New UnboundGridView(TryCast(grid, GridControl))
		End Function
	End Class
End Namespace

File: CS/E4434/CustomGrid/UnboundGridViewInfoRegistrator.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraGrid.Views.Base;
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Registrator;

namespace CustomGrid.CustomGrid {
    public class UnboundGridViewInfoRegistrator : GridInfoRegistrator {
        public override string ViewName {
            get {
                return "UnboundGridView";
            }
        }

        public override BaseView CreateView(GridControl grid) {
            return new UnboundGridView(grid as GridControl);
        }
    }
}
